api/src/main/java/org/openmrs/module/ugandaemrreports/reports/Helper.java

```diff
@@ -352,7 +452,7 @@ public static String getObsPeriod(Date period, Enums.Period periodType) {
                 .build();
     }
 
-    private static Connection getDatabaseConnection(Properties props) throws ClassNotFoundException, SQLException {
+    public static Connection getDatabaseConnection(Properties props) throws ClassNotFoundException, SQLException {
```

Comment:
@carapai This seems misplaced should it not be in a TestHelper class as its not used during running within OpenMRS environment

Reply:
This is used, uses openmrs to get mysql details at runtime